cli/providers/aws/translate.go

```diff
@@ -87,11 +93,53 @@ func loadASG(t *gocf.Template, s *service.Service, e *environment.Environment, a
 		UpdatePolicy:   updatePolicy,
 	}
 	t.Resources["ASG"] = asgResource
+
+	instanceIngressRules := gocf.EC2SecurityGroupRuleList{}
+	loadbalancerIngressRules := gocf.EC2SecurityGroupRuleList{}
+	for _, container := range s.Containers {
+		for _, port := range container.Ports {
```

Comment:
are the ports unique at this point? maybe we should flatten the entire list or something (e.g. what if two containers have the same port, would that matter for this?)